pkgs/applications/misc/spnavcfg/default.nix

```diff
@@ -1,33 +1,40 @@
-{ stdenv, lib, fetchFromGitHub, pkg-config, gtk2 }:
-
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  pkg-config,
+  libspnav,
+  qtbase,
```

Comment:
I'd prefer if this needs qt5 to have that be encoded in the package, not in `all-packages.nix`.